app.js

```diff
@@ -2,213 +2,65 @@
-        console.log(times[i] + sales[i]);
-        total += parseInt(sales[i]);
-      }
-    } return sales;
-  },
-  render: function() {
-    var ulEl = document.getElementById('firstPike');
-    var salesToday = this.cookieSales();
-    for (var i = 0; i < salesToday.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = times[i] + salesToday[i];
-      ulEl.appendChild(liEl);
-    }
-  }
-};
-firstPike.render();
+var customerTotal = 0;
+var cookiesTotal = 0;
 
-var seaTac = {
-  hourlyMin: 3,
-  hourlyMax: 24,
-  avgPersale: 1.2,
-  randomCustomers: function () {
-    var customers = Math.floor(Math.random() *
-          (this.hourlyMax - this.hourlyMin + 1) +
-          this.hourlyMin);
-    return customers;
-  },
-  cookiesPurchased: function() {
-    var cookies = Math.round(this.avgPersale * this.randomCustomers());
-    return cookies + ' cookies';
-  },
-  cookieSales: function() {
-    var sales = [];
-    var total = 0;
-    for(var i = 0; i < times.length; i++) {
-      if(i === (times.length - 1)) {
-        sales[i] = total + ' cookies';
-        console.log(times[i] + sales[i]);
-      } else {
-        sales[i] = this.cookiesPurchased();
-        console.log(times[i] + sales[i]);
-        total += parseInt(sales[i]);
-      }
-    } return sales;
-  },
-  render: function() {
-    var ulEl = document.getElementById('seaTac');
-    var salesToday = this.cookieSales();
-    for (var i = 0; i < salesToday.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = times[i] + salesToday[i];
-      ulEl.appendChild(liEl);
-    }
-  }
-};
-seaTac.render();
+var cookieStand = document.getElementById('cookieStand');
 
-var seaCenter = {
-  hourlyMin: 11,
-  hourlyMax: 38,
-  avgPersale: 3.7,
-  randomCustomers: function () {
-    var customers = Math.floor(Math.random() *
-            (this.hourlyMax - this.hourlyMin + 1) +
-            this.hourlyMin);
-    return customers;
-  },
-  cookiesPurchased: function() {
-    var cookies = Math.round(this.avgPersale * this.randomCustomers());
-    return cookies + ' cookies';
-  },
-  cookieSales: function() {
-    var sales = [];
-    var total = 0;
-    for(var i = 0; i < times.length; i++) {
-      if(i === (times.length - 1)) {
-        sales[i] = total + ' cookies';
-        console.log(times[i] + sales[i]);
-      } else {
-        sales[i] = this.cookiesPurchased();
-        console.log(times[i] + sales[i]);
-        total += parseInt(sales[i]);
-      }
-    } return sales;
-  },
-  render: function() {
-    var ulEl = document.getElementById('seaCenter');
-    var salesToday = this.cookieSales();
-    for (var i = 0; i < salesToday.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent = times[i] + salesToday[i];
-      ulEl.appendChild(liEl);
-    }
-  }
-};
-seaCenter.render();
+function CookieStore(storeName, hourlyMin, hourlyMax,cookieMin, cookieMax, avgPersale){
```

Comment:
You are not given a `cookieMin` and `cookieMax` by Pat, so where are these numbers coming from?